src/force-torque-estimator.cpp

```diff
@@ -459,6 +357,36 @@ namespace dynamicgraph
         getProfiler().stop(PROFILE_JOINTS_TORQUES_MOTOR_COMPUTATION);
         return s;
       }
+
+      /** Estimate the joints' positions, velocities and accelerations. */
+      DEFINE_SIGNAL_INNER_FUNCTION(q_dq_ddq, dynamicgraph::Vector)
```

Comment:
If we intend to make the new entity general purpose, I suggest we make ft_estimator dependent on the new entity. 

So the estimator would be taking the raw encoder signals as well as the smoothed and differentiated signals from the new entity. Would you agree?